fr.json

```diff
@@ -145,8 +145,81 @@
             "line1" : "Créer une tenue aléatoire avec tous les objets actuellement disponibles",
             "chooseText" : "Cliquez pour choisir",
             "randomise-all" : "Choix aléatoire"
+             },
```

Comment:
Wrong indentation. Please remove five spaces at the beginning of this line so that the closing bracket is at the correct level.